--- origsrc/gpa-0.7.5/src/hidewnd.c	2006-03-20 06:31:05.000000000 -0600
+++ src/gpa-0.7.5/src/hidewnd.c	2007-03-10 21:27:01.703125000 -0600
@@ -19,7 +19,7 @@
  */
 
 /* This module is only used in this environment */
-#if defined(__MINGW32__) || defined(__CYGWIN__)
+#if defined(__MINGW32__)
 #include <stdio.h>
 
 #include <windows.h>
--- origsrc/gpa-0.7.5/src/w32reg.c	2005-11-15 11:09:03.000000000 -0600
+++ src/gpa-0.7.5/src/w32reg.c	2007-03-10 21:27:01.703125000 -0600
@@ -20,7 +20,7 @@
  */
 
 #include <config.h>
-#if defined (_WIN32) || defined (__CYGWIN32__)
+#if defined (_WIN32)
  /* This module is only used in this environment */
 
 #include <stdio.h>
